hathor/transaction/transaction_metadata.py

```diff
@@ -36,7 +103,9 @@ class TransactionMetadata:
     accumulated_weight: float
     score: float
     first_block: Optional[bytes]
+    first_block_parent: Optional[bytes]
```

Comment:
What is it for? I feel we should remove it now because it's not being used anywhere in the PR.

Reply:
Yes, I'll remove this for now.

Reply:
Done.